app/src/main/res/layout/fragment_profile_menu.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    tools:context=".ProfileMenuFragment">
+
+    <LinearLayout
+        android:id="@+id/buttons"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
```

Comment:
You can do a recycleview, the recyvleview is more clean. You definie only one time the "structure" of the thing that can can be repeted

Reply:
I thought about it but:

"If your app needs to display a scrolling list of elements based on large data sets (or data that frequently changes), you should use RecyclerView as described on this page."

https://developer.android.com/guide/topics/ui/layout/recyclerview

this page pushed me to not use a recyclerview as it seems not to fit my need as my 4 menu buttons are not a large data set, won't change, and aren't scrolling

Reply:
okay, but can you put the title into string.xml, just to remain consistant ^^

Reply:
of course